app/views/admin/docs/_form.html.erb

```diff
@@ -99,14 +99,16 @@
   <% end %>
   <hr/>
   <h4 class="form-subhead"><%= t('additional_settings', default: 'Additional Settings') %>:</h4>
-  <div class= "form_group no-translate">
-    <label class="control-label" for="tag_list"><%= t('tags', default: 'Tags') %>: </label>
-    <%= f.text_field :tag_list, value: @doc.tag_list.to_s, class: 'selectize form-control' %>
-  </div>
+  <% if @doc.category_id != 1 %>
+    <div class= "form_group no-translate">
+      <label class="control-label" for="tag_list"><%= t('tags', default: 'Tags') %>: </label>
+      <%= f.text_field :tag_list, value: @doc.tag_list.to_s, class: 'selectize form-control' %>
+    </div>
 
-  <%= f.input :meta_description, label: I18n.translate("activerecord.attributes.doc.meta_description", locale: I18n.locale), value: @doc.read_translated_attribute(:meta_description)  %>
-  <%= f.input :title_tag, label: I18n.translate("activerecord.attributes.doc.title_tag", locale: I18n.locale), value: @doc.read_translated_attribute(:title_tag)  %>
-  <%= f.input :keywords, label: I18n.translate("activerecord.attributes.doc.keywords", locale: I18n.locale), value: @doc.read_translated_attribute(:keywords)  %>
+    <%= f.input :meta_description, label: I18n.translate("activerecord.attributes.doc.meta_description", locale: I18n.locale), value: @doc.read_translated_attribute(:meta_description)  %>
+    <%= f.input :title_tag, label: I18n.translate("activerecord.attributes.doc.title_tag", locale: I18n.locale), value: @doc.read_translated_attribute(:title_tag)  %>
+    <%= f.input :keywords, label: I18n.translate("activerecord.attributes.doc.keywords", locale: I18n.locale), value: @doc.read_translated_attribute(:keywords)  %>
+  <% end %>
```

Comment:
Awesome, I think we should also exclude the `feature on front page` and `allow comments` since those are likely to be confusing...